usage/dist/src/main/license/LICENSE

```diff
@@ -0,0 +1,1691 @@
+
+Permission to use, copy, and distribute the contents of this
+document, or the W3C document from which this statement is linked,
+in any medium for any purpose and without fee or royalty is hereby
+granted, provided that you include the following on ALL
+copies of the document, or portions thereof, that you use:
+
+A link or URL to the original W3C document.
+
+The pre-existing copyright notice of the original author, or if
+it doesn't exist, a notice of the form: "Copyright © [$date-of-document] World Wide Web
+Consortium, (Massachusetts
+Institute of Technology, Institut National de Recherche en Informatique et en
+Automatique, Keio
+University). All Rights Reserved.
+http://www.w3.org/Consortium/Legal/" (Hypertext is preferred, but a
+textual representation is permitted.)
+
+If it exists, the STATUS of the W3C document.
+
+When space permits, inclusion of the full text of this NOTICE
+should be provided. We request that authorship
+attribution be provided in any software, documents, or other items
+or products that you create pursuant to the implementation of the
+contents of this document, or any portion thereof.
+
+No right to create modifications or derivatives of W3C documents
+is granted pursuant to this license. However, if additional
+requirements (documented in the Copyright
+FAQ) are satisfied, the right to create modifications or
+derivatives is sometimes granted by the W3C to individuals
+complying with those requirements.
+
+THIS DOCUMENT IS PROVIDED "AS IS," AND COPYRIGHT HOLDERS MAKE NO
+REPRESENTATIONS OR WARRANTIES, EXPRESS OR IMPLIED, INCLUDING, BUT
+NOT LIMITED TO, WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+PARTICULAR PURPOSE, NON-INFRINGEMENT, OR TITLE; THAT THE CONTENTS
+OF THE DOCUMENT ARE SUITABLE FOR ANY PURPOSE; NOR THAT THE
+IMPLEMENTATION OF SUCH CONTENTS WILL NOT INFRINGE ANY THIRD PARTY
+PATENTS, COPYRIGHTS, TRADEMARKS OR OTHER RIGHTS.
+COPYRIGHT HOLDERS WILL NOT BE LIABLE FOR ANY DIRECT, INDIRECT,
+SPECIAL OR CONSEQUENTIAL DAMAGES ARISING OUT OF ANY USE OF THE
+DOCUMENT OR THE PERFORMANCE OR IMPLEMENTATION OF THE CONTENTS
+THEREOF.
+
+The name and trademarks of copyright holders may NOT be used in
+advertising or publicity pertaining to this document or its
+contents without specific, written prior permission. Title to
+copyright in this document will at all times remain with copyright
+holders.
+
+-----------------------------------------------------------------------------
+
+xmlpull-xmlpull-1.1.3.1.jar
+
+All of the XMLPULL API source code, compiled code, and documentation
+contained in this distribution *except* for tests (see separate LICENSE_TESTS.txt)
+are in the Public Domain.
+
+XMLPULL API comes with NO WARRANTY or guarantee of fitness for any purpose.
+
+Initial authors:
+
+  Stefan Haustein
+  Aleksander Slominski
+
+-----------------------------------------------------------------------------
+
+xpp3-xpp3_min-1.1.4c.jar
+
+Indiana University Extreme! Lab Software License
+
+Version 1.1.1
```

Comment:
I'm guessing that 1.1.1 refers to the version of the license, which is almost always versioned differently to the software (think GPLv2 and GPLv3)